CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ title: Changelog
 * `<cc-input-text>`: add `inline` prop to place the label on the left of the `<input>` element. Add new `inline` story to show this behavior.
 * `<cc-input-number>`: add `inline` prop to place the label on the left of the `<input>` element. Add new `inline` story to show this behavior.
 * `<cc-select>`: add `inline` prop to place the label on the left of the `<select>` element. Add new `inline` story to show this behavior.
+* `<cc-toggle>`: add `inline` prop to place the label on the left of the group of radio input elements. Add new `inline` story to show this behavior.
```

Comment:
Ignore the whole commit right now. I'd rather discuss this in sync as the `<fieldset>` / `<legend>` styling is quite tricky :roll_eyes: